### PR TITLE
fix bug

### LINKED ISSUE #17: Docs not opening in Windows Local
There is a bug happening, when trying to access to Docs via local host, the app is crashing. Maybe is because the unsafe use of `src.doc` 

Environment:

Windows -> Localhost

### PR DESCRIPTION
**Pull request by Hicham2012**

### CODE REVIEW

File: src/app/docs/page.tsx [hunk 1]
```diff
@@ -51,10 +51,11 @@ const DocsPage = () => {
   };
   const content = organizePathsBySections(paths);
 
+
   return (
     <DocApp
       markdownContent={markdownContent}
-      docStructure={content.src.docs}
+      docStructure={content}
```
Comment (tkalejandro) [inline src/app/docs/page.tsx]: Hey, are you sure this doesnt break the other links.
All links are been auto generated inside  the content.src.docs (console.log)
The DocApp component requires that list to generate all the info.

I tried to use the docs in the deployed version and all links lead to /docs . Can you try ?

**Review by tkalejandro — CHANGES_REQUESTED** [review]

File: src/app/docs/page.tsx (hunk 1 above)
Comment (Hicham2012) [reply src/app/docs/page.tsx]: Good catch, I fixed the issue.
Can you give it another review?

File: src/modules/DocApp/DocApp.tsx [hunk 2]
```diff
@@ -30,9 +30,15 @@ const DocApp = ({ markdownContent, docStructure, paths, directory }: DocAppProps
...
+
           return (
             <li key={key}>
-              <Link href={finalUrl.endsWith(key) ? finalUrl : ''}>{key}</Link>
+              {/*
+                This will add the url to paths
+                while removing 'src/' from paths
+                to make it accessible
+               */}
+              <Link href={finalUrl + key.slice(3)}>{key}</Link>
```
Comment (tkalejandro) [inline src/modules/DocApp/DocApp.tsx]: I think you a problem with this logic:  The routes paths stll broken 

Some examples:

http://localhost:3000/docs/github/how-to-start-a-new-branch.md-to-start-a-new-branch.md
http://localhost:3000/docs/prettier/current-rules.mdrent-rules.md
http://localhost:3000/docs/three/audio.mdio.md

Proabably is the slice or something else.

**Review by tkalejandro — CHANGES_REQUESTED** [review]

File: src/modules/DocApp/DocApp.tsx
Comment (Hicham2012) [inline src/modules/DocApp/DocApp.tsx:1]: Can you test it again on localhost @tkalejandro ?
Also, I noticed that in Vercel they only show the last part of the link (the .md file) when hovering the content.

Comment (tkalejandro) [reply src/modules/DocApp/DocApp.tsx:1]: The paths still broken :-( 


http://localhost:3000/how-to-start-a-new-branch.md

for example should be:

http://localhost:3000/docs/github/how-to-start-a-new-branch.md


--

I think the solution is more related to the bug at the beginning passing content.src.docs
Is not safe typed.

Just to tell: cont.src.docs is not the same as just content, If you wanna use conent, then use src.docs in the DocApp component.

The problem with this component is probably TS doenst complain and think the initial logic. 

Just try one more time if doenst work out, I can have a look later, and maybe safe type that, and maybe will work on windows.

For @AlbertLeipzig  , he mentioned he actually dont have problem to see the Docs. Can you confirm this Albert?

File: src/modules/DocApp/DocApp.tsx [hunk 3]
```diff
@@ -30,9 +30,12 @@ const DocApp = ({ markdownContent, docStructure, paths, directory }: DocAppProps
       <ul>
         {keys.map((key) => {
           const finalUrl = assignUrl(key) ?? '';
+          let newUrl = key.replace('src', '')
```
Comment (tkalejandro) [inline src/modules/DocApp/DocApp.tsx]:
```suggestion
          const newUrl = key.replace('src', '')
```

File: src/modules/DocApp/DocApp.tsx [hunk 4]
```diff
@@ -30,9 +30,12 @@ const DocApp = ({ markdownContent, docStructure, paths, directory }: DocAppProps
...
           return (
             <li key={key}>
-              <Link href={finalUrl.endsWith(key) ? finalUrl : ''}>{key}</Link>
+              {/* Remove 'src' from path */}
+              {/* <Link href={finalUrl.endsWith(key) ? finalUrl : key.replace('src', '')}>{key}</Link> */}
+              <Link href={newUrl}>{key}</Link>
               {/* This will loop the code until nested is finished */}
               {generateNestedList(content[key] as ContentSection)}
```
Comment (tkalejandro) [inline src/modules/DocApp/DocApp.tsx]:
```suggestion
              {generateNestedList(content.src.docs)} // not sure if u have to use "as ContentSection" again.
```

**Review by tkalejandro — CHANGES_REQUESTED** [review]
Dont hate XD

**Review by tkalejandro — APPROVED** [review]
Nioce!

**Pull request merged**